problems/loops.js

```diff
@@ -82,29 +156,53 @@ function tillXTimes9WhileLoop() {}
 * 4 * 9 = 36
 * 5 * 9 = 45
  */
-function tillXTimes9ForLoop() {}
-
+// function tillXTimes9ForLoop() {}
+const tillXTimes9ForLoop = (x) => {
+  let arr = [];
+  for (let num = 0; num <= x; num++) {
+    arr.push(num * 9);
+  }
+  return arr;
+};
 
 /**
  * Returns an array with all the numbers that end in 5 from 1 to 100
 * Must be done with a while loop
  * @returns {number[]} [5, 15, 25, ..., 95]
 
  */
-function endsWithFiveWhileLoop() {}
-
-
+//function endsWithFiveWhileLoop() {}
+const endsWithFiveWhileLoop = () => {
+  let arr = [];
+  let num = 1;
+  while (num <= 100) {
+    if (num % 10 === 5) {
+      arr.push(num);
+      num++;
```

Comment:
move increment out, and have no else. 
Do we have to increment by 1? Is there something clever we can do that also gets rid  of the if condition?